qbraid_qir/qasm3/maps.py

```diff
@@ -487,17 +105,7 @@ def map_qasm_op_to_pyqir_callable(op_name: str) -> tuple[Callable, int]:
     ]
 
     for mapping, qubits in qasm_op_mappings:
-        if op_name in mapping:
-            return mapping[op_name], qubits
+        if op_name.lower() in mapping:
+            return mapping[op_name.lower()], qubits
```

Comment:
```suggestion
        op_name_lower = op_name.lower()
        if op_name_lower in mapping:
            return mapping[op_name_lower], qubits
```

avoid redundant computation